prediction_market_agent/utils.py

```diff
@@ -33,7 +34,7 @@ def serp_api_key(self) -> SecretStr:
         )
 
     @property
-    def openai_api_key(self) -> SecretStr:
+    def openai_api_key(self) -> SecretStrV1:
```

Comment:
Please keep the `SecretStr` here and use https://github.com/gnosis/prediction-market-agent-tooling/blob/bac961f73de2b0dce8b0892a484834dadf3683db/prediction_market_agent_tooling/gtypes.py#L78 where necessary.

V1 is older, so make sense to use that one only with older libraries that aren't updated.

```suggestion
    def openai_api_key(self) -> SecretStr:
```